spec/lib/github_service/commands/remove_reviewer_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+RSpec.describe GithubService::Commands::RemoveReviewer do
+  subject { described_class.new(issue) }
+  let(:issue) { double(:fq_repo_name => "org/repo") }
+  let(:command_issuer) { "nickname" }
+
+  before do
+    allow(GithubService).to receive(:valid_assignee?).with("org/repo", "listed_user") { true }
+    allow(GithubService).to receive(:valid_assignee?).with("org/repo", "good_user") { true }
+    allow(GithubService).to receive(:valid_assignee?).with("org/repo", "bad_user") { false }
+
+    allow(subject).to receive(:requested_reviewers).and_return(["listed_user"])
+  end
+
+  after do
+    subject.execute!(:issuer => command_issuer, :value => command_value)
+  end
```

Comment:
Why is this necessary? What are you cleaning up here?

Reply:
It is used also in [assign_spec.rb](https://github.com/ManageIQ/miq_bot/blob/master/spec/lib/github_service/commands/assign_spec.rb#L14), [add_reviewer_spec.rb](https://github.com/ManageIQ/miq_bot/blob/master/spec/lib/github_service/commands/add_reviewer_spec.rb#L14), [add_label_spec](https://github.com/ManageIQ/miq_bot/blob/master/spec/lib/github_service/commands/add_label_spec.rb#L18), [remove_label_spec.rb](https://github.com/ManageIQ/miq_bot/blob/master/spec/lib/github_service/commands/remove_label_spec.rb#L10). I took inspiration from these mentioned specs before and on this basis I created my remove_reviewer_spec.rb.

Reply:
It's weird, but all of the others do their thing-under-test in an after block.  I don't like it, but for this PR, it's consistent.